tests/test_autotuner.py

```diff
@@ -1,16 +1,10 @@
-from __future__ import absolute_import
 from functools import reduce
 from operator import mul
-try:
-    from StringIO import StringIO
-except ImportError:
-    # Python3 compatibility
-    from io import StringIO
 
 import pytest
 import numpy as np
-from devito import Grid, Function, TimeFunction, Eq, Operator, configuration, silencio
-from devito.logger import logger, logging
+from devito import (Grid, Function, TimeFunction, Eq, Operator, configuration,
+                    silencio, ruido)
```

Comment:
silencio, ruido ?

Reply:
they are wrappers to locally change the logger and profile levels. I think it's just Spanish ? not sure why

Reply:
We can replace them in English if possible....